cli/exchange/node.go

```diff
@@ -86,7 +86,7 @@ func NodeCreate(org, nodeIdTok, node, token, userPw, email string, arch string,
 	exchUrlBase := cliutils.GetExchangeUrl()
 
 	// Assume the user exists and try to create the node, but handle the error cases
-	putNodeReq := exchange.PutDeviceRequest{Token: nodeToken, Name: nodeName, SoftwareVersions: make(map[string]string), PublicKey: []byte(""), Arch: arch} // we only need to set the token
+	putNodeReq := exchange.PutDeviceRequest{Token: nodeToken, Name: nodeName, NodeType: nodeType, SoftwareVersions: make(map[string]string), PublicKey: []byte(""), Arch: arch} // we only need to set the token
```

Comment:
Can the user use the node create command to change the type of a node? They should not be allowed to do that.

Reply:
I do not understand. This is the node create command. why can't user specify the node type?

Reply:
They can specify it, the question is can they change the nodetype of an existing node with this command? I think they can, which I think is a problem.

Reply:
changed to not allow node type change if the node eixtst.